webapp/src/main/webapp/templates/freemarker/body/partials/individual/propStatement-dataDefault.ftl

```diff
@@ -15,7 +15,13 @@
 <@showStatement statement property datatype />
 
 <#macro showStatement statement property datatype>
+
     <#assign theValue = statement.value />
+
+<#-- create a weblink if the property range is a URI AND it contains http -->
+    <#if property.rangeDatatypeURI?? && theValue?contains("http")>
```

Comment:
Use starts_with instead of contains so that it only matches where the protocol is http